public/stylesheets/setting.css

```
@@ -0,0 +1,55 @@
+body {
+  padding: 0;
+  margin: 0;
+  width: 100%;
+  height: 100dvh;
+  background-image: url(../images/background-image.png);
+  background-size: cover;
+  background-position: center;
+}
+
+.settingItem {
+  /* ドラッグによる文字選択禁止 */
+  -ms-user-select: none;
+  -webkit-user-select: none;
+  user-select: none;
+
+  font-size: 50px;
+  position: relative;
+  bottom: 70px;
+}
```

ここも命名をケバブケースに揃えてください。
それと`position: relative;`や`bottom`はいらないです。

```suggestion
.setting-item {
  /* ドラッグによる文字選択禁止 */
  -ms-user-select: none;
  -webkit-user-select: none;
  user-select: none;

  font-size: 2rem;
}
```